src/tinylogger/decorator.py

```
@@ -0,0 +1,60 @@
+    :param args: The positional arguments passed to the function.
+    :param kwargs: The keyword arguements passed to the function.
+    :return: A dictionary of parameter names mapped to their values.
+    """
+    try:
+        # Bind the positional and keyword arguments to the function signature
+        bound_args = inspect.signature(func).bind(*args, **kwargs)
+        # Apply the default for any missing arguements
+        bound_args.apply_defaults()
+        # Arguments is an OrderedDict, we convert it into plain dict
+        return dict(bound_args.arguments)
+    except Exception:
```

Ruff (0.14.2)

Do not catch blind exception: `Exception`

(BLE001)